scripts/plotting/postfitPlots.py

```diff
@@ -807,7 +832,10 @@ def make_plots(
     procs = [
         k.replace("expproc_", "").replace(f"_{fittype};1", "")
         for k in fitresult.keys()
-        if fittype in k and k.startswith("expproc_") and "hybrid" not in k
+        if fittype in k
+        and k.startswith("expproc_")
+        and "hybrid" not in k
```

Comment:
what if "and all(x not in k for x in ["hybrid", "masked"]) " to be prepared for additional conditions, and also to save one line